llvm-remote-index.md

```diff
@@ -0,0 +1,79 @@
+# LLVM remote index service
+
+We plan to provide [remote index](/remote-index.md) service for LLVM
+developers, this document describes the details of this service.
+
+We provide our best effort support for remote index service but it is not an
+official product by either LLVM or Google.
+
+**NOTE**: The service is currently in the alpha testing stage and we would
+like to gradually expand the scope of availability and feature set.
+
+## Overview
+
+LLVM remote index service is implemented by the team maintaining clangd. We
```

Comment:
s/implemented/provided

Reply:
Also rather than saying `we` and talking about future, I would suggest saying it directly. `The service is provided to make development of LLVM on less powerful machines easier ...`